checkUrlExists/lambda/app.js

```diff
@@ -0,0 +1,47 @@
+// const axios = require('axios')
```

Comment:
What were these two lines for?

Reply:
The new lambda was generated with SAM CLI...i have quite a bit of default stuff left in